README.md

```diff
@@ -12,58 +12,72 @@ Heroku Connect Toolbelt Plugin
 
 # Commands
 
-    heroku connect:auth                   - Authenticate a connection to Salesforce
-    heroku connect:preauth                - Store a Salesforce authentication for future use
-    heroku connect:export                 - Export a mapping configuration JSON file
-    heroku connect:import                 - Import a mapping configuration JSON file
-    heroku connect:info                   - Display connection information 
-    heroku connect:mapping:state MAPPING  - Returns the state of a mapping
-    heroku connect:setup                  - Configure a new connection
-    heroku connect:state                  - Return the state flag for a single connection
-    heroku connect:pause                  - Pause a connection
-    heroku connect:resume                 - Resume a connection
-    heroku connect:restart                - Restart a connection
+    heroku connect:auth                    - Authenticate a connection to Salesforce
+    heroku connect:export                  - Export a mapping configuration JSON file
+    heroku connect:import FILE             - Import a mapping configuration JSON file
+    heroku connect:info                    - Display connection information 
+    heroku connect:mapping:state MAPPING   - Return the state of a mapping
+    heroku connect:mapping:delete MAPPING  - Delete an existing mapping
+    heroku connect:mapping:reload MAPPING  - Reload a mapping's data from Salesforce
+    heroku connect:pause                   - Pause a connection
+    heroku connect:resume                  - Resume a connection
+    heroku connect:restart                 - Restart a connection
+    heroku connect:setup                   - Configure a new connection
+    heroku connect:state                   - Return the state flag for a single connection
 
 # Examples
 
-Download the mapping configuration
+Download an existing mapping configuration
 
-    heroku connect:export
+    $ heroku connect:export
+    Saved config-file: app-name-resource-name.json
 
 # Tutorial
 
-Create an authorization into your Salesforce org. Note the result token.
+Make sure you have a Heroku app, with a Postgres database attached
 
-    heroku connect:preauth
+## Add the Heroku Connect add-on to your app
 
-    Token: xxx
+    $ heroku addons:create herokuconnect
 
+## Link the new connection (the Heroku Connect add-on instance) to your Heroku user
 
-Make sure you have a Heroku app, with a Postgres database attached
+    $ heroku connect:info
+
+## Now link the connection to the database, specifying the config var and schema name
+
+    $ heroku connect:setup -d DATABASE_URL -s salesforce
```

Comment:
Looking a bit further, these are in fact optional currently. The code here will default to `DATABASE_URL` and `salesforce`. But given that it would do so with no prior indication of this, nor any kind of confirmation, I'm not sure that's the best idea. I see two options:

1. We make them required arguments instead of optional flags

2. We leave them optional on the command, but rather than silently provide default values, we interactively prompt for values. We could easily ask for a schema name and default to salesforce if the user presses enter, and we could query the API for a list of available config vars, just like we do on the web.